.github/ISSUE_TEMPLATE/transfer-repo-to-this-organization.yaml

```
@@ -0,0 +1,82 @@
+name: Transfer repository to this organization
+description: Request to transfer a repository you own to this organization
+title: "[Transfer]: Repo-name"
```

we usually don't use title tags:
```suggestion
title: "Repo-name"
```